src/components/pages/Breakdown.vue

```diff
@@ -427,6 +451,39 @@ export default {
         })
     },
 
+    processCSVFile (file, config) {
+      return new Promise((resolve, reject) => {
+        Papa.parse(file, {
+          config: config,
+          error: reject,
+          complete: (results) => {
+            resolve(results.data)
+          }
+        })
+      })
+    },
+
+    previewImportFile (formData) {
+      this.loading.importing = true
+      this.errors.importing = false
+      this.formData = formData
+      const file = formData.get('file')
+      this.processCSVFile(file)
+        .then((results) => {
+          this.parsedCSV = results
+          this.hideImportModal()
+          this.loading.importing = false
+          this.showPreviewModal()
+        })
+    },
+
+    reuploadFile () {
```

Comment:
The function name doesn't seem to match the behavior. It looks more like a reset function.